base/docs/Docs.jl

```diff
@@ -675,4 +675,19 @@ function hasdoc(binding::Docs.Binding, sig::Type = Union{})
 end
 
 
+"""
+    undocumented_names(mod::Module; all=false)
+
+Return an array of undocumented symbols in `module` (that is, lacking docstrings).
+`all=false` returns only exported symbols; whereas `all=true` also includes
+non-exported symbols, following the behavior of [`names`](@ref).
```

Comment:
This, along with the `names` docstring, should mention that the names are sorted in ascending order (by `[isless](@ref)`, i.e. alphabetically).